libraries/rust-buildtime/context/test

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+exec cargo test
```

Comment:
fyi - test doesn't actually function yet 😬 also have a ticket open for that but thanks for adding it in here

Reply:
:+1: - acknowledged as reasonable and resolving this comment.